### PR TITLE
Update bug_report.md

### PR DESCRIPTION
**Pull request by echo-juliet**
Copy edits to the default GitHub bug report template based on discussion @ refactoring meeting 2/19.

# Description

# Test Plan

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,38 +1,33 @@
...
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser [e.g. stock browser, safari]
- - Version [e.g. 22]
+**Technical Details:**
+These are important!
+ - OS & version: [e.g. Catalina 10.15.7]
+ - Browser [e.g. chrome, safari, check "about" in your browser menu for version info]
+ - SC Instance [e.g. SC, 1Hive, etc]
```
Comment (wchargin) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: nit: Maybe expand “SC” to “SourceCred” here for clarity?

**Review by wchargin — APPROVED** [review]
LGTM, with one suggestion inline.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (topocount) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: Two additional notes here: 
* Do we want to specifically request the browser version?
* I think we should also request the link to the instance on line 30.

**Review by topocount — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (echo-juliet) [reply .github/ISSUE_TEMPLATE/bug_report.md:30]: Good points, integrating these.

**Pull request merged**